### ISSUE TITLE
Windows: Failure: executable "mypy.exe" not found

### ISSUE THREAD

**Issue by leinardi** [leinardi/mypy-pycharm#12]
### Step 1: Are you in the right place?
- [x] I have verified there are no duplicate active or recent bugs, questions, or requests
- [x] I have verified that I am using the latest version of the plugin.

### Step 2: Describe your environment
 - Plugin version: `0.10.0`
 - PyCharm/IDEA version: `2018.2.2 Community`
 - Mypy version: `0.630`
 - OS: Windows 10

### Step 3: Describe the problem:
![image](https://user-images.githubusercontent.com/273338/45814579-5206ca00-bcd6-11e8-9627-f33242cf2e6d.png)

When you hit the test button:

```
10:48 AM    Mypy Plugin: Failure: executable "H:\Miniconda3\envs\dev36\Scripts\mypy.exe" not found
```

From a cmd window I can prove the mypy executable exists (via installing mypy via pip) and runs:

```
H:\Miniconda3\envs\dev36\Scripts>.\mypy.exe
usage: mypy [-h] [-v] [-V] [more options; see below]
            [-m MODULE] [-p PACKAGE] [-c PROGRAM_TEXT] [files ...]
mypy: error: Missing target module, package, files, or command.
```
The above directory is on the path, and I can run mypy from any directory on a command window and it will give the same output as above.

_(original issue reported on [stackoverflow](https://stackoverflow.com/questions/52416011/pycharm-mypy-plugin-not-working-but-looks-to-be-installed-correctly))_

**Comment by leinardi** [comment]
This is mostly an OS specific issue. Currently I don't have access to a Windows machine so I am not able to debug this issue.
It would be great if someone with Windows could help me by testing some development builds, that would speed up the fixing since I should get access to a Windows machine only the next Monday.

I made a new development build where I am just logging some errors and debug info of the method `isMypyPathValid()`: [mypy-plugin-0.10.0.zip](https://github.com/leinardi/mypy-pycharm/files/2402235/mypy-plugin-0.10.0.zip)

You should:

1. remove the old Mypy plugin (just to be sure)
2. install the [new build](https://github.com/leinardi/mypy-pycharm/files/2402235/mypy-plugin-0.10.0.zip) (it has the same version code of the current live one)
3. open the settings, add the path to the file "mypy.exe" and press the test button
4. send me the idea.log file (https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files)

**Comment by iGiant** [comment]
[idea.log](https://github.com/leinardi/mypy-pycharm/files/2404950/idea.log)

**Comment by leinardi** [comment]
Hi @iGiant, thanks for the log file. I have a possible fix for the issue, could you please try this new build and let me know if fixes the problem? [mypy-plugin-0.10.0.zip](https://github.com/leinardi/mypy-pycharm/files/2405724/mypy-plugin-0.10.0.zip)

I also tried to implement an auto-detection of mypy for Windows, if you can please check if the `mypy.exe` is auto-detected when you leave the relative field empty inside the plugins settings:
![image](https://user-images.githubusercontent.com/273338/45887059-0b3ed000-bdbb-11e8-9b3a-1d3031ecc801.png)

**Comment by iGiant** [comment]
I deleted the previous version, then put the proposed one. When you start pycharm, an error occurs. The plugin finds the path to mypy.exe. I attach a log (several starts)
[idea.log](https://github.com/leinardi/mypy-pycharm/files/2405813/idea.log)

**Comment by leinardi** [comment]
Thanks for the tests, I have a new version that should fix the errors: [mypy-plugin-0.10.0.zip](https://github.com/leinardi/mypy-pycharm/files/2405928/mypy-plugin-0.10.0.zip)

Beside the error messages, is the plugin working properly (both real-time and on-demand scanning)? Have you noticed anything wrong?
